### PR TITLE
RST-6576 Document download bug fix and details page to be displayed on misc option selection

### PR DESCRIPTION
**Pull request by eyss1234**

### CODE REVIEW

File: app/presenters/summary/html_sections/miam_exemptions.rb [hunk 1]
```diff
@@ -40,7 +40,7 @@ def answers
           ),
           FreeTextAnswer.new(
             :exemption_reasons, c100.exemption_reasons,
-            change_path: edit_steps_miam_exemptions_exemption_reasons_path
+            change_path: edit_steps_miam_exemptions_exemption_reasons_path,
```
Comment (zaparka) [inline app/presenters/summary/html_sections/miam_exemptions.rb:43]: Why did you add this ?

File: features/miam.feature [hunk 2]
```diff
@@ -68,35 +68,35 @@ Feature: MIAM journey
...
-    And I click the "Continue" button
-
-    Then I should see "You don’t have to attend a MIAM"
-    Then I should see "Other exemptions"
-    Then I should see "There is no authorised family mediator with an office within 15 miles of your home"
-
-    And I click the "Continue" link
-    Then I should see "Safety concerns"
-    And the mediation changes end
+#  Scenario: Applicant did not attend a MIAM and has selected a misc valid reason
```
Comment (zaparka) [inline features/miam.feature:71]: I assume you comment this out because these test are not working and there is no way around it? Something Kalvin mentioned.

**Review by lambley — APPROVED** [review]
